app/views/layouts/application.html.erb

```diff
@@ -16,7 +16,7 @@
     <main class="px4">
       <div class="flash-message">
         <% flash.each do |name, msg| %>
-          <%= content_tag :div, msg, class: name %>
+          <%= content_tag :div, msg.html_safe, class: name %>
```

Comment:
This was necessary to added to flash messages so that links would display correctly